docs/src/app/components/pages/components/Menu/ExampleSecondary.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Menu from 'material-ui/lib/menus/menu';
+import MenuItem from 'material-ui/lib/menus/menu-item';
+import Divider from 'material-ui/lib/divider';
+import ArrowDropRight from 'material-ui/lib/svg-icons/navigation-arrow-drop-right';
+
+const arrow = (
+  <ArrowDropRight />
+);
```

Comment:
We can leave it here or move it to the render function.
I don't know what's best. Their no performance implication either way.
@alitaheri Though?